parent/src/pom-template.xml

```diff
@@ -352,11 +352,12 @@
                     <groupId>org.openrewrite.maven</groupId>
                     <artifactId>rewrite-maven-plugin</artifactId>
                     <version>${openrewrite-maven-plugin.version}</version>
-                    <configuration>
-                        <!-- OpenRewrite configuration here -->
-                    </configuration>
                     <dependencies>
-                        <!-- OpenRewrite dependencies here -->
+                        <dependency>
+                            <groupId>io.micronaut.openrewrite</groupId>
+                            <artifactId>micronaut-openrewrite</artifactId>
+                            <version>LATEST</version>
```

Comment:
This is not ok. You need to treat `micronaut-openrewrite` as any other module, importing its bom, and then referencing the corresponding version property here.

Reply:
we have a meeting with @alvarosanchez yesterday and we are going to pass the coordinates directly to the open rewrite maven plugin invocation via system property.